test/testgroups/operandrequest_group.go

```diff
@@ -134,14 +202,101 @@ func TestOperandRequestCRUD(t *testing.T) {
 	assert.NoError(err)
 	assert.Len(reg.Status.OperatorsStatus["jenkins"].ReconcileRequests, 2, "the reconcile request number should be equal 2 for operator jenkins")
```

Comment:
May I ask that if we delete `jenkins` operator from both OperandRequest, or there is only one `jenkins` and we delete it, how could we check the OperatorStatus?

Will it be `ReconcileRequests == 0` or The key "jenkins" does not exist in OperatorStatus anymore?

Reply:
Then it won't exist in the status of OperandRegistry.

Reply:
> Then it won't exist in the status of OperandRegistry.

Got it, thank you!